CONTRIBUTING.md

```diff
@@ -4,3 +4,7 @@
 
 :+1::tada: First off, I'm really glad you're reading this, because we need volunteer developers to help with the development of fAIr! :tada::+1:
 We welcome and encourage contributors of all skill levels and we are committed to making sure your participation in our tech collective is inclusive, enjoyable and rewarding. If you have never contributed to an open-source project before, we are a good place to start and will make sure you are supported every step of the way. If you have **any** questions, please ask!
+
+## Documentation contributions 
+
+Create pull requests (PRs) for changes that you think are needed to the documentation of fAIr.As of now you can find the documentation work at the [docs](https://github.com/hotosm/fAIr/tree/master/docs) directory.
```

Comment:
Use relative URL links instead of obsolete so that it will work on anywhere . like this (../docs )

Reply:
See [Relative links and image paths in README files](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files).

Reply:
Resolved by https://github.com/hotosm/fAIr/pull/202/commits/80a5d995177a43c911ed1418a590f51d9a05e0f0